packages/graphback-runtime/src/createRuntimeServices.ts

```diff
@@ -0,0 +1,49 @@
+import { GraphQLObjectType, GraphQLSchema } from "graphql"
+import { PubSubEngine } from 'graphql-subscriptions';
+import { GraphbackDataProvider } from './data/GraphbackDataProvider';
+import { CRUDService } from './service/CRUDService';
+import { GraphbackModelConfig } from "./GraphbackModelConfig";
+
+export type RuntimeServicesOptions<Provider extends GraphbackDataProvider, Service extends CRUDService> = {
+  schema: GraphQLSchema,
+  models: GraphbackModelConfig[]
+  db: any
```

Comment:
If we pass db then we do not need provider and also if we pass provider then we do not need db